plugins/ocp/ocp-print-json.c

```diff
@@ -829,13 +823,12 @@ static void json_c7_log(struct nvme_dev *dev, struct tcg_configuration_log *log_
 				  le32_to_cpu(log_data->pro_rlc));
 	json_object_add_value_int(root, "TCG Error Count", le32_to_cpu(log_data->tcg_ec));
 
-	memset((__u8 *)res, 0, 458);
-	for (j = 0; j < 458; j++)
-		res += sprintf(res, "%d", log_data->rsvd3[j]);
-	json_object_add_value_string(root, "Reserved3", res_arr);
```

Comment:
Not happy that the json output changes. Adding is not an issue but changing is causing regressions. Though I think adding the reserved fields into json is wrong. But this ship has already sailed.